packages/devtools/lib/src/memory/memory.dart

```diff
@@ -144,6 +181,8 @@ class MemoryScreen extends Screen with SetStateMixin {
       memoryChart.pause();
     });
 
+    history = div(c: 'history-navigation-hidden');
```

Comment:
nit:
prefer
'history-navigation hidden'
so that you have a separate class for being history-navigation and a class for the state of hidden vs shown.

Reply:
Done.